python/ray/util/client/server/server.py

```diff
@@ -556,7 +584,11 @@ def default_connect_handler(job_config: JobConfig = None):
         logs_servicer=logs_servicer,
         grpc_server=server,
     )
-    server.start()
+    loop = asyncio.get_event_loop()
+    loop.run_until_complete(server.start())
```

Comment:
oof mixing event loop between threads can lead to hard to debug issues.

Reply:
resolved offline. But will use separate asyncio event loop.